src/Raven.Server/Documents/Handlers/Debugging/StorageHandler.cs

```diff
@@ -380,14 +380,17 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
         {
             bool first = true;
 
-            writer.WriteStartObject();
+            if (envs.Count() > 1)
```

Comment:
It means then that WriteReport should be per env, and code inside this `if` should be handled outside

Reply:
Done